internal/utils/test/mock/terminal_ui.go

```diff
@@ -38,81 +39,85 @@ var (
 	StaticTime = time.Date(1989, 6, 22, 1, 23, 45, 0, time.UTC)
 )
 
-type ui struct {
+// UI is a mocked terminal.UI
+type UI struct {
 	terminal.UI
+
+	AskOneFn func(answer interface{}, prompt survey.Prompt) error
```

Comment:
I know we discussed this offline before, but remind me again where/why this was needed?

Reply:
This was so I could mock the AskOne() and have it spit out an error. I didn't see any examples in the code base so far.

Reply:
Example test case is on line 154 of inputs_test.go

Reply:
I'd rather we remove this mock ui for now.  I actually haven't written any tests that simulate errors coming from `survey` (as you can tell).  we might be able to do the same with some simulated key press (ctrl+c or something) and avoid the mock entirely.  thoughts?

Reply:
Yeah if we are able to simulate or pass signals, I think pattern is a better fit for these tests.